docs/00-introduction.md

```diff
@@ -0,0 +1,75 @@
+    - [GitLens - Git supercharged (Eric Amodio)](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens)
+    - [Markdown All in One (Yu Zhang)](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one)
+    - [markdownlint (David Anson)](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint)
+- **Git client** (including Git Bash on Windows). A graphical Git client like Gitkraken is optional (and *not* recommended).
+- **Docker** (including Docker Compose). You can install Docker Desktop for Windows or Mac, or Docker CE for Linux. Use WSL2 as the backend on Windows.
+
+It is best to install the required software using a package manager, e.g., [Chocolatey](http://chocolatey.org/) on Windows or [Homebrew](https://brew.sh/) for Mac. Linux users are, of course, familiar with this method and use their distribution's package manager, e.g., `apt` (Debian, Ubuntu), `dnf` (Fedora and RHEL family), `pacman` (Arch Linux), etc. Check if you have these tools installed. If so, upgrade to the latest versions. If not, install them through the package manager.
+
+:information_source: For this course, the use of GitHub Copilot is allowed.
+
+Hier is de vertaling van de HTML-pagina naar het Engels en omgezet in Markdown:
+
+## Git configuration
+
+All your work for this course is tracked in the version control system Git. On the banner of the intro page of the Chamilo course, you'll find a link that creates a new repository where you save all your work. This repository is visible to you and the instructor. In addition to configuring the systems you set up, you will also maintain your documentation there, such as lab reports and cheat sheets.
+
+Guidelines for starting your Git project:
+
+- You should already have a GitHub account. If you haven't done so yet, be sure to link your `@student.hogent.be` email address to the account. You can then apply for the [**GitHub Student Developer Pack**](https://education.github.com/pack) which includes various interesting offers. It's possible to register multiple email addresses, and it makes sense to also link your private address. This way, you can continue using your GitHub account after graduation.
+
+- Create an [**SSH key pair**](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) to simplify pushing to GitHub, follow the instructions provided. For convenience, enter an empty passphrase (otherwise, you'll need to enter the passphrase every time you use the key).
+
+- Enable [**two-factor authentication**](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa) on your GitHub account. This is a security measure that is now required on GitHub.
+
+- **Basic Git configuration**, if you haven't done this before (check in the configuration file `~/.gitconfig`):
```

Comment:
The XDG specifications specify `$HOME/.config/` as the place to be for config files: https://wiki.archlinux.org/title/XDG_Base_Directory.
E.g. for git the config file would be `$HOME/.config/git/config`.